src/test/java/io/cdap/plugin/successfactors/source/SuccessFactorsSourceTest.java

```diff
@@ -91,6 +92,7 @@ public void testConfigurePipelineWithInvalidUrl() throws Exception {
       .entityName("entity name")
       .username("username")
       .password("password")
+      .authType("basicAuth")
```

Comment:
It seems we are not using the builder object the way we should.
`pluginConfigBuilder` is already initialised in the `setUp` and in the respective tests we just need to update or add the relevant values instead of reassigning the values again. eg.

Inside `setUp`
```
pluginConfigBuilder = SuccessFactorsPluginConfig.builder()
      .referenceName("unit-test-ref-name")
      .baseURL("http://localhost")
      .entityName("entity name")
      .username("username")
      .password("password")
      .authType("basicAuth")
      .selectOption("col1,col2,   \n  parent/col1,\r       col3     ");
```
And if we need to add `paginationType` & `filterOption` in any `tests`, then it can be done like
```
pluginConfigBuilder.paginationType("serverSide").filterOption("$topeq2");
```
and just call the `build()`.